### PR TITLE
Custom Prefix Support

### PR DESCRIPTION
**Pull request by OctopusProteins**
Adding support to change the command prefix
ex. /help, .help, ./help....

### CODE REVIEW

File: octoprint_discordremote/command.py [hunk 1]
```diff
@@ -94,9 +94,10 @@ def timelapse(self):
     def help(self):
         builder = EmbedBuilder()
         builder.set_title('Commands, Parameters and Description')
+        # builder.set_description("All commands should be prefaced with the bot prefix: '%s'" % self.plugin.get_settings().get(["prefix"]))
```
Comment (cameroncros) [inline octoprint_discordremote/command.py]: @OctopusProteins Did you mean to leave this uncommented?

Comment (OctopusProteins) [reply octoprint_discordremote/command.py]: Yes, it was remnant from when I was going to leave the commands without their prefix on the help list. You can remove it.

Comment (cameroncros) [reply octoprint_discordremote/command.py]: Okay, is it okay if I leave it to you to remove it? Also, the unittests have been broken, please fix them. I believe you need to properly mock the calls to "self.plugin.get_settings().get(["prefix"])"

Something like (In unittests/test_command.py):
    _settings = mock.Mock()
    _settings.get = mock.Mock()
    _settings.get.return_value = "/"

Comment (cameroncros) [reply octoprint_discordremote/command.py]: Okay, I'll do it, and i'll send the commit your way so you can have a look.

Comment (OctopusProteins) [reply octoprint_discordremote/command.py]: Awesome, thanks!

**Comment by cameroncros** [comment]
@OctopusProteins Checkout: https://github.com/cameroncros/OctoPrint-DiscordRemote/commit/f82c60dbf674212f5e73c57ecd538098b3419a44

**Pull request merged**